fuzz/make_corpus.py

```diff
@@ -0,0 +1,23 @@
+import json
+import sys
+import os.path
+
+"""
```

Comment:
For reference, Google has a style guide for some popular languages, including [Python](https://google.github.io/styleguide/pyguide.html#382-modules). The link shows an example documentation for files.